server/routes/apiRoutes.js

```diff
@@ -12,83 +12,154 @@ var db = require("../models");
 module.exports = function (app) {
 
 
-
-  app.get("/api/user/:email", function (req, res) {
-    db.User.findOne({
+  //this route should not need to be modified
+  //not sure what this is supposed to be used for though....need clarification
+  app.get("/api/user/:email", function (req, res) 
+  {
+    db.user.findOne(
+      {
       where: { email: req.params.email }
-
-    }).then(function (dbUser) {
-      res.json(dbUser);
+      })
+    .then(function (dbuser) 
+    {
+      res.json(dbuser);
     });
   });
 
-  app.post("/api/user", function (req, res) {
+  //create a user account
+  app.post("/api/user", function (req, res) 
+  {
     console.log(req.body)
-    db.User.create({
-      email: req.body.userEmail,
-      password: req.body.userPassword
+    db.user.create(
+      {
+      fname: req.body.fname,
```

Comment:
What if the `req` doesn't have a body? Like an empty `POST`. This will blow up spectacularly. Consider some validation here:
``` 
if(req.body) {
   // code
} else {
 // throw a 400
}